src/FffGcodeWriter.cpp

```diff
@@ -1517,24 +1515,13 @@ bool FffGcodeWriter::processSkinPart(const SliceDataStorage& storage, LayerPlan&
                 }
             }
         }
-
-        // determine inner_skin_outline
-        if (skin_part.insets.size() > 0)
-        {
-            inner_skin_outline = &skin_part.insets.back();
-            offset_from_inner_skin_outline = -mesh_config.insetX_config.getLineWidth() / 2;
-        }
-    }
-
-    if (inner_skin_outline == nullptr)
-    {
-        inner_skin_outline = &skin_part.outline;
     }
 
     int extra_infill_shift = 0;
+    coord_t offset_from_inner_skin_infill = 0;
```

Comment:
I think a variable name like `offset` could be more clear. You'd just see "oh, there is no offset" because it says `offset = 0` right above it.

(Not blocking.)